services/nsn/modules/nodejs/nsn-get-routing/src/processor/nsn-routing-create-processor.ts

```diff
@@ -13,14 +13,29 @@ export const postNsn = async (event: APIGatewayProxyEvent, context: Context): Pr
     }
     console.log('1 body - ' + event.body);
 
-    const { routing_id, owa, is_civ_mgr, is_mil_mgr, ric, created_by } = JSON.parse(event.body);
-
+    let { routing_id, owa, is_civ_mgr, is_mil_mgr, ric, created_by } = JSON.parse(event.body);
+    
```

Comment:
Is defining const put some restrictions ? I mean, I am not sure making the variable declaration here with 'let' here.